### DISCUSSION TITLE
Можно ли спользовать useForm вместе с imaskjs ??

### DISCUSSION THREAD

**Discussion by HeisPovedim** [uNmAnNeR/imaskjs#719] [Q&A] [answered]

**Comment by karandashovrz** [comment, marked as answer]
@HeisPovedim  Sure, why not? Do you mean useForm from react-hook-form? You need to use onAccept callback property in IMaskInput and call onChange callback from controller wrapper inside onAccept. You can't use onChange handler directly in IMaskInput, it doesnt work.
